src/useFind.ts

```diff
@@ -112,10 +116,39 @@ export default <CustomApplication extends Application>(feathers: CustomApplicati
       }
 
       try {
+        const originalParams: Params = params.value;
+        const originalQuery: Query & { $limit?: number } = originalParams.query || {};
         // TODO: the typecast below is necessary due to the prerelease state of feathers v5. The problem there is
         // that the AdapterService interface is not yet updated and is not compatible with the ServiceMethods interface.
-        const res = await (service as unknown as ServiceMethods<M>).find(params.value);
-        data.value = Array.isArray(res) ? res : [res];
+        const res = await (service as unknown as ServiceMethods<M> | AdapterService<M>).find(originalParams);
+        if (call !== currentFindCall.value) {
+          return;
+        }
+        if (isPaginated(res)) {
+          let loadedPage: Paginated<M> = res;
+          let loadedItemsCount = loadedPage.data.length;
+          const limit: number = originalQuery.$limit || loadedPage.data.length;
+          data.value = [...loadedPage.data];
+          while (!unloaded && loadedPage.total > loadedItemsCount) {
```

Comment:
@lukashass  Not really. If you make a request with pagination, the backend should tell you how many items are available accross all pages fitting to your query. They send it to you via the total field and then you have your stop criteria.

Or what situation do you have in mind with loading all items?

Reply:
@mariusheine I get the point of @lukashass. It somehow seems to be an anti-pattern. You normally introduce / use pagination as loading and using all data at once is to much for your client / server. The question IMO is why do you even need all that data to be stored in a reactive variable at once and wouldn't a normal `feathers.find` do the trick? You could also disable pagination by the params option or query directly with a specific page.

Reply:
@anbraten I need to query a lot of data which are only send to me as small pages from a third party system due to background limitations e.g. AWS Lambda with their chunk size. SO I can not disable it. 

And the issue with the normal feathers find is that I am not reactive to some live updates.

Reply:
@mariusheine The important distinction here is _pagination_ vs _chunking_. They are different concepts IMHO.

Reply:
@lukashass Ok so what would you suggest? I could try to create an internal wrapper inside the project...

Reply:
Thinking about it a bit more, maybe lambda actually uses chunking synonymous to pagination. But loading everything in chunks is definitely something else 😉 

@mariusheine Anyways, since this code appears to be a workaround for lambda missing a "load everything" option, I think this should be part of the Lambda feathers service implementation.

Of course, it would be even better to actually implement pagination in the frontend (UI), improve the pagination handling in `use-feathers` for real and not load everything at once 🙈

Reply:
@lukashass @anbraten Alright. Then why not keeping this feature and rename it to "chunking"?

Reply:
@lukashass But yes. I got the difference between pagination and chunking. And yes you were right at the top: We load all the items matching the given query

Reply:
@anbraten Yeah there I totally agree. That would be quite crucial. So I will try to add an according option if I want chunking or pagination.

Reply:
@anbraten @lukashass I adjusted the title and added the chunking option. What do you think about it now?